### ISSUE TITLE
Use of undeclared crate or module `yake_rust`

### ISSUE THREAD

**Issue by dhra** [quesurifn/yake-rust#1]
I'm trying to import yake_rust by the following command:
```bash
cargo add yake_rust
```

My `Cargo.toml` file:
```toml
[dependencies]
yake-rust = "0.1.1"
```

The `main.rs` file:
```rust
use std::time::Instant;
use yake_rust::Yake;
use yake_rust::ResultItem;

fn main() {
    let now = Instant::now();

    let text = r#"
    Google is acquiring data science community Kaggle. Sources tell us that Google is acquiring Kaggle, a platform that hosts data science and machine learning 
    competitions. Details about the transaction remain somewhat vague, but given that Google is hosting its Cloud 
    Next conference in San Francisco this week, the official announcement could come as early as tomorrow. 
    Reached by phone, Kaggle co-founder CEO Anthony Goldbloom declined to deny that the acquisition is happening. 
    Google itself declined 'to comment on rumors'. Kaggle, which has about half a million data scientists on its platform, 
    was founded by Goldbloom  and Ben Hamner in 2010. 
    The service got an early start and even though it has a few competitors like DrivenData, TopCoder and HackerRank, 
    it has managed to stay well ahead of them by focusing on its specific niche. 
    The service is basically the de facto home for running data science and machine learning competitions. 
    With Kaggle, Google is buying one of the largest and most active communities for data scientists - and with that, 
    it will get increased mindshare in this community, too (though it already has plenty of that thanks to Tensorflow 
    and other projects). Kaggle has a bit of a history with Google, too, but that's pretty recent. Earlier this month, 
    Google and Kaggle teamed up to host a $100,000 machine learning competition around classifying YouTube videos. 
    That competition had some deep integrations with the Google Cloud Platform, too. Our understanding is that Google 
    will keep the service running - likely under its current name. While the acquisition is probably more about 
    Kaggle's community than technology, Kaggle did build some interesting tools for hosting its competition 
    and 'kernels', too. On Kaggle, kernels are basically the source code for analyzing data sets and developers can 
    share this code on the platform (the company previously called them 'scripts'). 
    Like similar competition-centric sites, Kaggle also runs a job board, too. It's unclear what Google will do with 
    that part of the service. According to Crunchbase, Kaggle raised $12.5 million (though PitchBook says it's $12.75) 
    since its   launch in 2010. Investors in Kaggle include Index Ventures, SV Angel, Max Levchin, Naval Ravikant,
    Google chief economist Hal Varian, Khosla Ventures and Yuri Milner 
    "#;

    let kwds = Yake::new(None, None, None, None).get_n_best(text.to_string(), Some(30));

    println!("{:?}", kwds);
    let elapsed = now.elapsed();
    println!("Elapsed: {:.2?}", elapsed); 
}
```

Output:
```terminal
error[E0432]: unresolved import `yake_rust`
  --> src/main.rs:
   |
22 | use yake_rust::Yake;
   |     ^^^^^^^^^ use of undeclared crate or module `yake_rust`

error[E0432]: unresolved import `yake_rust`
  --> src/main.rs:
   |
23 | use yake_rust::ResultItem;
   |     ^^^^^^^^^ use of undeclared crate or module `yake_rust`

```

**Comment by quesurifn** [comment]
Thanks for opening this issue. I'll have a look this weekend.

**Comment by quesurifn** [comment]
Thanks for pointing this out. 

There were two issues causing this:

- incorrect project structure for a crate
- incompatible lib option turned on in Cargo.toml

It's been fixed in 0.1.6 published to crates.io
